src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -409,7 +409,7 @@ private async Task<(Flow, State, StateTrace, Stopwatch)> RedirectToSubflowAsync(
             context.Flow = subflow;
             var newState = subflow.States.Single(s => s.Root);
 
-            await _stateSessionManager.SetStateAsync(userIdentity, shortNameOfSubflow, cancellationToken);
```

Comment:
Is this method still being used? If not remove its implementation

Reply:
It's was a ethod that already existed...